packages/palette/src/themes/v3.tsx

```diff
@@ -42,54 +39,54 @@ const BREAKPOINTS_SCALE = Object.assign(
  * design system from our design team
  */
 export const THEME = {
-  id: "v3",
+  // Backfill with v2
+  ...THEME_V2,
```

Comment:
Aw no, what happened?

Reply:
I'm gonna try rolling this back; I thought I was having a type issue with the theme but it turned out do to me using `as const` on the theme to convert them to readonly (which let me do a typeguard on id).